commons-dlp/src/main/java/org/exoplatform/commons/dlp/service/impl/DlpPositiveItemServiceImpl.java

```diff
@@ -83,6 +94,12 @@ public Long getDlpPositiveItemsCount() {
     return dlpPositiveItemDAO.count();
   }
 
+  @Override
+  public String getDlpKeywords() {
+    SettingValue<?> settingValue = settingService.get(Context.GLOBAL, Scope.GLOBAL.id("DlpKeywords"), EXO_DLP_KEYWORDS);
+    return settingValue != null ? settingValue.getValue().toString() : System.getProperty(DLP_KEYWORDS);
+  }
+  
```

Comment:
Here we have duplicated code. getDlpKeyword is present in DlpServiceConnector  and here.
I think it should be moved to DlpService which is the main service for DLP
DlpPositiveItemService is for managing DlpPositivieItems